components/SkilledImpact/elements/SkillContent.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, Switch } from 'react-native';
+import styles from '../../../constants/SkilledImpact/OrgSkilledImpactBody';
+import Sync from '../../../assets/jsicons/bottomnavigation/Sync';
+import ChevronBottom from '../../../assets/jsicons/miscIcons/ChevronBottom';
+import ChevronRight from '../../../assets/jsicons/miscIcons/ChevronRight';
+
+class SkillContent extends React.Component {
+  constructor(props) {
+    //TODO props edits and states
+    super(props);
+    this.state = {
+      skillExpand: true,
+      isReachable: props.isAcceptingHelp,
+      skills: props.skills
+    };
+  }
+
+  skillExpand = () =>{
+    this.setState({skillExpand: !this.state.skillExpand})
+  };
+
+  toggleReachMeSwitch = () =>{
+    //TODO: api request to change the switch state.
+  };
```

Comment:
**suggestion**: `toggleAcceptingHelpRequests()`, and we do have backend support for this already (not sure if you were planning on finishing in this PR or another). You should be able to `PUT /users/:id` with `{"accepting_help_requests":true}`